.github/workflows/master.yml

```diff
@@ -278,9 +278,6 @@ jobs:
       - name: Run tests
         run: |
           pytest --splits=${{ matrix.splits }} --group=${{ matrix.group }} tests/evaluate --ignore=tests/evaluate/test_default_evaluator_delta.py
-      - name: Run tests with delta
-        run: |
-          pytest tests/evaluate/test_default_evaluator_delta.py
```

Comment:
can we make the following change in `test_default_evaluator_delta.py`?

```diff
diff --git a/tests/evaluate/test_default_evaluator_delta.py b/tests/evaluate/test_default_evaluator_delta.py
index 46b6c797a1..8d67a07b49 100644
--- a/tests/evaluate/test_default_evaluator_delta.py
+++ b/tests/evaluate/test_default_evaluator_delta.py
@@ -34,8 +34,12 @@ def test_write_to_delta_fails_without_spark():
             )
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def spark_session_with_delta():
+    # Kill any existing Spark session to avoid conflicts.
+    if session := SparkSession.getActiveSession():
+        session.stop()
+
     with tempfile.TemporaryDirectory() as tmpdir:
         with (
             SparkSession.builder.master("local[*]")
```